examples/service-catalog/service-catalog.yaml

```diff
@@ -366,6 +366,8 @@ objects:
           - "5m"
           - --feature-gates
           - OriginatingIdentity=true
+          - --feature-gates
```

Comment:
I believe that this flag is a list that has to be passed.  However, we decided that we weren't going to add a flag to enable this in oc cluster up - so I think that you can close this out.